docs/networking/diagnostics/inspecting-network-connections-with-netstat/index.md

```diff
@@ -368,221 +729,50 @@ eth0            1      ff02::fb
-tcp6       0      0 [::]:http               [::]:*                  LISTEN      root       35577141
-tcp6       0      0 [::]:ssh                [::]:*                  LISTEN      root       35572961
-tcp6       0      0 [::]:https              [::]:*                  LISTEN      root       35577145
-{{< /output >}}
-
-### Displaying the PID and the Owner of processes
-
-The `-p` option displays the Process ID and the owner of a process. However, `netstat(8)` requires root privileges for this.
+The following command calculates the total number of listening processes per username:
 
-    netstat -lt -p
-    {{< output >}}
-Active Internet connections (only servers)
-Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
-tcp        0      0 localhost:mysql         0.0.0.0:*               LISTEN      24555/mysqld
-tcp        0      0 0.0.0.0:ssh             0.0.0.0:*               LISTEN      1008/sshd
-tcp      101      0 0.0.0.0:smtp            0.0.0.0:*               LISTEN      8576/master
-tcp6       0      0 [::]:http               [::]:*                  LISTEN      1808/apache2
-tcp6       0      0 [::]:ssh                [::]:*                  LISTEN      1008/sshd
-tcp6       0      0 [::]:https              [::]:*                  LISTEN      1808/apache2
-{{< /output >}}
+    sudo netstat -lte | awk '{print $7}' | grep -v Address | grep -v "^$" | sort | uniq -c | awk '{print $2 ": " $1}'
 
-### Combining `-p` and `-e`
-
-Combining `-p` with `-e` while having root privileges will generate the following kind of output:
-
-    netstat -lt -p -e
-    {{< output >}}
-Active Internet connections (only servers)
-Proto Recv-Q Send-Q Local Address           Foreign Address         State       User       Inode      PID/Program name
-tcp        0      0 localhost:mysql         0.0.0.0:*               LISTEN      mysql      35862475   24555/mysqld
-tcp        0      0 0.0.0.0:ssh             0.0.0.0:*               LISTEN      root       35572959   1008/sshd
-tcp      101      0 0.0.0.0:smtp            0.0.0.0:*               LISTEN      root       35544149   8576/master
-tcp6       0      0 [::]:http               [::]:*                  LISTEN      root       35577141   1808/apache2
-tcp6       0      0 [::]:ssh                [::]:*                  LISTEN      root       35572961   1008/sshd
-tcp6       0      0 [::]:https              [::]:*                  LISTEN      root       35577145   1808/apache2
+{{< output >}}
+mysql: 1
+root: 5
 {{< /output >}}
 
-The previous command displays extended information about the listening TCP processes of the current Linux machine.
-
-## Using AWK to process netstat output
-
-The AWK programming language can help you process `netstat(8)` output and generate handy reports.
+- The `netstat` command collects the listening TCP connections and includes the users for the connections' processes in the output.
```

Comment:
Restated command explanation as a list, and reordered to so that the explanation followed the order of the commands in the sequence